woocommerce/woocommerce
# Add product tabs to product layout

### All Submissions:






### Changes proposed in this Pull Request:

Adds in the product tabs to the header and moves sections to the respective tabs.

Note that this PR does not yet fully address extensibility, which will be saved for a future cycle.

<img width="643" alt="Screen Shot 2022-12-06 at 11 53 38 AM" src="https://user-images.githubusercontent.com/10561050/206011597-7bbc8cce-ff12-4ea3-be79-6599ee051184.png">


Closes #35706 .






### How to test the changes in this Pull Request:



1. Navigate to the new product management experience (`Products -> Add new (MVP)`)
2. Click the various tabs in the header to navigate between product sections (General/Pricing/etc)
3. Make sure that all content is shown and works as expected
4. Narrow your viewport to make sure that the header placement works with various screen sizes
5. Scroll down inside a given tab and then click another tab to make sure you're scrolled back to the top



### Other information:




### FOR PR REVIEWER ONLY:
